From: "Matthew Wilcox (Oracle)" <willy@infradead.org>
Subject: nilfs2: remove nilfs_page_get_nth_block
Date: Mon, 16 Oct 2023 21:11:02 +0100

All users have now been converted to get_nth_block().

Link: https://lkml.kernel.org/r/20231016201114.1928083-16-willy@infradead.org
Signed-off-by: Matthew Wilcox (Oracle) <willy@infradead.org>
Acked-by: Ryusuke Konishi <konishi.ryusuke@gmail.com>
Cc: Andreas Gruenbacher <agruenba@redhat.com>
Cc: Pankaj Raghav <p.raghav@samsung.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 fs/nilfs2/page.h |    6 ------
 1 file changed, 6 deletions(-)

--- a/fs/nilfs2/page.h~nilfs2-remove-nilfs_page_get_nth_block
+++ a/fs/nilfs2/page.h
@@ -52,10 +52,4 @@ unsigned long nilfs_find_uncommitted_ext
 #define NILFS_PAGE_BUG(page, m, a...) \
 	do { nilfs_page_bug(page); BUG(); } while (0)
 
-static inline struct buffer_head *
-nilfs_page_get_nth_block(struct page *page, unsigned int count)
-{
-	return get_nth_bh(page_buffers(page), count);
-}
-
 #endif /* _NILFS_PAGE_H */
_
